docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -159,6 +159,27 @@ for instance:
     - if you have extended `CountryController` revise your changes – `new` and `edit` actions were added
 - if you have extended `Localization` class, you have to add type-hints to extended methods because they were added in the parent class ([#806](https://github.com/shopsys/shopsys/pull/806))
     - if you have extended method `Localization::getAdminLocale()` only to have administration in a different language than english, you can delete it and set parameter `shopsys.admin_locale` in your `parameters.yml` file instead
+- changed JS validation of forms in popup and login form in popup is now get by AJAX ([#782](https://github.com/shopsys/shopsys/pull/782))
+    - in `window.js` add options `textHeading = ''` and `cssClassHeading: ''` to `var defaults` and then add it to `$windowContent` and add div with `js-validation-errors` class to every popup.
```

Comment:
I would change this to: 
This javascript files has been changed, please update them appropriately:
- src/Shopsys/ShopBundle/Resources/scripts/frontend/login.js
- src/Shopsys/ShopBundle/Resources/scripts/frontend/window.js

For me is better to see diff on Github than this instructions

Reply:
The change or login.js is mentioned in the optional point below and It is only few lines so I think it's more readable here

Reply:
We have discussed it and you want to make it more straightforward